plugins/modules/swim_workflow_manager.py

```diff
@@ -1890,7 +1890,8 @@ def get_diff_activation(self):
             self.result['changed'] = True
             self.status = "success"
             self.partial_successful_activation = True
-            self.msg = "Image with Id '{0}' activated and partially successful.".format(image_id)
+            self.msg = ("Image with ID '{0}' was activated, but only partially successful. The image activation failed for the "
+                        "following device(s): {1}.").format(image_id, "', '".join(device_ips_list))
             self.log("For Device(s) {0} Image activation gets Failed".format(str(device_ips_list)), "CRITICAL")
```

Comment:
self.log("Image activation failed for the following device(s): {0}".format(", ".join(device_ips_list)), "CRITICAL")